packages/eslint-plugin-prairielearn/src/rules/no-unused-sql-blocks.ts

```diff
@@ -0,0 +1,65 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+import { ESLintUtils } from '@typescript-eslint/utils';
+
+function extractSqlBlockDefinitions(sqlContent: string): Set<string> {
+  const regex = / *-- *BLOCK +([^ \n]+) */g;
```

Comment:
This regex is the same as the one in the postgres package, but I exclude newlines as well.